homeassistant/components/generic_hygrostat/strings.json

```diff
@@ -4,7 +4,7 @@
     "step": {
       "user": {
         "title": "Add generic hygrostat",
-        "description": "Create a entity that control the humidity via a switch and sensor.",
+        "description": "Create a humidifier entity that control the humidity via a switch and sensor.",
```

Comment:
This entity can both humidify and dehumidify. So this seems wrong.

Reply:
Still, it creates an humidifier entity... as it is the entity component platform...

So this adjustment is correct (but IMHO not solving the issue in the issues either).

Reply:
Is the issue clarity? I was trying to come up with a clearer description, how's this?

"Create a humidifier entity that uses sensor readings to monitor and manage humidity levels, controlling a switch connected to either a humidifier or dehumidifier."

It's a little more wordy, I'll check how it looks in the UI. 

I'm unsure how important it is for the description to mention the switch and sensor but genetic_thermostat does, so I left it in.